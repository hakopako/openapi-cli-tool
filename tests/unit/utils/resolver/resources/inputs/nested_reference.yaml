application/json:
  schema:
    $ref: '#/components/schemas/Pet'
application/x-www-form-urlencoded:
  schema:
    $ref: '#/components/schemas/PetForm'
text/plain:
  schema:
    type: string

components:
  schemas:
    Pet:
      $ref : '#/components/schemas/PetForm'
    PetForm:
      type: object
      properties:
        name:
          type: string
        fav_number:
          type: string
      required:
        - name
        - fav_number
      additionalProperties: false
