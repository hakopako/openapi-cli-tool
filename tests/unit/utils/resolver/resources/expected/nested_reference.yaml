application/json:
  schema:
    type: object
    properties:
      name:
        type: string
      fav_number:
        type: string
    required:
      - name
      - fav_number
    additionalProperties: false
application/x-www-form-urlencoded:
  schema:
    type: object
    properties:
      name:
        type: string
      fav_number:
        type: string
    required:
      - name
      - fav_number
    additionalProperties: false
text/plain:
  schema:
    type: string

components:
  schemas:
    Pet:
      type: object
      properties:
        name:
          type: string
        fav_number:
          type: string
      required:
        - name
        - fav_number
      additionalProperties: false
    PetForm:
      type: object
      properties:
        name:
          type: string
        fav_number:
          type: string
      required:
        - name
        - fav_number
      additionalProperties: false
